components/shared/BasemapSelector.vue

```diff
@@ -87,7 +87,24 @@ const emitBasemapChange = () => {
       :style="{ top: topPosition }"
       @click="toggleBasemapWindow"
     >
-      <img src="/map.svg" alt="Map Icon" />
+      <NuxtImg
+        src="/map.svg"
+        alt="Map Icon"
+        preset="icon"
+        loading="eager"
+        :custom="true"
+        v-slot="{ src, isLoaded, imgAttrs }"
+      >
+        <!-- Show the actual icon when loaded -->
+        <img v-if="isLoaded" v-bind="imgAttrs" :src="src" />
+
+        <!-- Show a placeholder while loading -->
+        <img
+          v-else
+          src="https://placehold.co/30x30/cccccc/666666?text=🗺️"
+          alt="Map icon placeholder"
+        />
+      </NuxtImg>
```

Comment:
Doesn't render 
![image](https://github.com/user-attachments/assets/3c6df270-0be7-48bf-adb8-a81a6ece61bd)

Can `<NuxtImg>` actually be used for svgs?